app/src/main/java/com/example/mystore/repository/IProductsRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.mystore.repository
+
+import com.example.mystore.model.ProductModel
+import retrofit2.Response
+
+interface IProductsRepository {
```

Comment:
Note: `I....` is more of a Java convention. Not a big deal but you can just call it `ProductsRepository` and then the implementation - `ProductsRepositoryImpl` or something.